### PR TITLE
section for third party / community plugins

### PR DESCRIPTION
**Pull request by SeanFeldman**
Initial stab at third party / community contributed plugins.
As per comment [here](https://github.com/Azure/azure-service-bus-dotnet/issues/141#issuecomment-313249934) will require review and rewording to comply with legal.
/cc @nemakam

### CODE REVIEW

**Comment by nemakam** [comment]
Sounds good. But keeping this on hold till we finalize on legal and compliance requirements.

**Comment by SeanFeldman** [comment]
>  But keeping this on hold till we finalize on legal and compliance requirements.

Why to overload our brains? Put a ~~ring~~ [label](https://github.com/Azure/azure-service-bus-dotnet/labels/on%20hold) on it 😃 

[image](https://www.youtube.com/watch?v=-RP19fnff_c)

**Comment by nemakam** [comment]
@SeanFeldman , Could you resolve the conflicts here so that we can go ahead with the merge?

**Comment by SeanFeldman** [comment]
@nemakam resolved

File: README.md [hunk 1]
```diff
@@ -25,3 +25,9 @@ The folling plugins are included with this repository:
 
 * [Key Vault](./src/Microsoft.Azure.ServiceBus.KeyVaultPlugin/readme.md)
 * [Message ID](./src/Microsoft.Azure.ServiceBus.MessageIdPlugin/readme.md)
+
+## Third party provided plugins
+
+* [Claim Check pattern](https://www.nuget.org/packages/ServiceBus.AttachmentPlugin)
```
Comment (nemakam) [inline README.md]: Sorry for the late comment. I feel this list should start after the notice.

Comment (nemakam) [reply README.md]: @SeanFeldman could you take care of this small change

Comment (nemakam) [reply README.md]: Also, curious. Do you want to link it to your GitHub repo instead of nuget package?

Comment (SeanFeldman) [reply README.md]: No problem. I didn't know what would be the preference. From the nuget package you can get to the project home (GitHub) as well. But I'll change for consistency.

Comment (SeanFeldman) [reply README.md]: Done @nemakam

**Review by nemakam — APPROVED** [review]

**Pull request merged**